packages/marshal/src/passStyleOf.js

```diff
@@ -195,7 +199,10 @@ export const passStyleOf = makePassStyleOf([
   CopyRecordHelper,
   TaggedHelper,
   RemotableHelper,
+  // TODO: Move ErrorHelper above RemotableHelper
+  // so an Error with no own keys is not rejected as a bad remotable.
   ErrorHelper,
+  PromiseHelper,
```

Comment:
Postponing this TODO would be fine. But this PR does move it in `HelperTable`. I prefer these stay in the same order, so do it in both or neither. I prefer both.

Reply:
Done: https://github.com/endojs/endo/pull/1313/commits/3dfbfbeacfe8b38bc92b69fd2ce663b03f9697a5